packages/datahub-data/migrations/0001_create_object_item.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `object_item` (
+  `id` serial AUTO_INCREMENT NOT NULL,
+  `object_id` varchar(256) NOT NULL,
```

Comment:
Is this where the identifier of the object will be stored (i.e. the IRI)?

Reply:
Yes. Would you like me to rename it to make it more clear? Do you prefer `object_iri`?

Reply:
The rest of the code also calls it an ID. For example, in the function name and response of heritageObjects.getById.

Reply:
Or is the problem the max of `256`?

Reply:
Decided in meeting that type `text` is better for `object_id`